docs/pages/build-reference/apk.mdx

```diff
@@ -8,16 +8,17 @@ The default file format used when building Android apps with EAS Build is an [An
 
 ### Managed projects
 
-By default, EAS Build produces Android App Bundle, you can change it by modifying the [eas.json](/build/eas-json.mdx) file:
+By default, EAS Build produces Android App Bundle, you can change it by modifying the [eas.json](/build/eas-json.mdx) file by either:
 
-- setting `buildType` to `apk`
-- setting `developmentClient` to true
+- setting `buildType` to `apk` or
+- setting `developmentClient` to `true` or
 - setting `gradleCommand` to `:app:assembleRelease`, `:app:assembleDebug` or any other gradle command that produces APK
```

Comment:
Nice.

Should you offer suggestions about a preferred way to do it?

I guess this gets into a wider discussion of why there are 3 ways to do it in the first place ;-)

Which way is recommended?

Reply:
> Should you offer suggestions about a preferred way to do it?
I think, yes. It can also specify the `developmentClient` as the default way:

```markdown
- `developmentClient` to `true` (**default**)
```

> I guess this gets into a wider discussion of why there are 3 ways to do it in the first place ;-)

It was a design decision but that's a great point. I'll share your feedback on this and talk to the team as to why the design for generating APKs is this way. If there is a specific reason why there are multiple options, I think we can update this doc to reflect that.

> Which way is recommended?

For new Expo projects `developmentClient` is the recommend way since it is automatically added to **eas.json** file when `eas build:configure` command is run.

Reply:
Perfect.